.github/workflows/ci_docs.yaml

```diff
@@ -4,8 +4,8 @@ on:
     branches:
       - main
     paths:  # run this action only when the docs folder is changed
-      - 'docs/hr/content/**'
-      - 'docs/hr/*.js'
+      - 'docs/content/**'
+      - 'docs/*.js'
```

Comment:
I am not sure if this is an estimate that temporarily does not update the documentation, so it has not been modified accordingly. If it is, I will cancel this change. @blythed

Reply:
Yes good point.